index.bs

```diff
@@ -551,34 +642,42 @@ spec:css-syntax-3;
 
   <div algorithm="relevant credential interfaces">
     The <dfn for="CredentialRequestOptions">relevant credential interface objects</dfn> for a given
-    {{CredentialRequestOptions}} (|options|) is a set of [=interface objects=], collected as follows:
+    {{CredentialCreationOptions}} or {{CredentialRequestOptions}} (|options|) is a set of [=interface objects=], collected as follows:
 
     <ol class="algorithm">
-      1.  Let |settings| be the <a>current settings object</a>
+        1.  Let |settings| be the <a>current settings object</a>
+
+        1.  Let |interface objects| be the set of [=interface objects=] on |settings|'
+            [=environment settings object/global object=].
+
+        1.  Let |relevant interface objects| be an [=set/empty=] [=set=].
+
+        1.  For each |object| in |interface objects|:
 
-      2.  Let |interface objects| be the set of [=interface objects=] on |settings|'
-          [=environment settings object/global object=].
-         
-      3.  Let |relevant interface objects| be an empty set.
-      
-      4.  For each |object| in |interface objects|:
+            1.  If |object|'s [=inherited interfaces=] do not <a for="set">contain</a> {{Credential}},
+                <a for="iteration">continue</a>.
 
-          1.  If |object|'s [=inherited interfaces=] do not <a for="set">contain</a> {{Credential}},
-              <a for="iteration">continue</a>.
+            1.  Let |credentialType| be |object|'s {{Credential/[[type]]}} slot's value.
 
-          2.  Let |key| be |object|'s {{Credential/[[type]]}} slot's value.
+            1.  Let |key| be the result of looking up the |credentialType|'s |options|'s
+                [=dictionary member|member=] [=identifier=] in the registry of
+                [[#sctn-registry-cred-type-to-options-member-identifier|Credential Type to Options Member Identifier Mappings]].
 
-          3.  If |options|[|key|] <a for="map">exists</a>, <a for="set">append</a> |object| to
-              |relevant interface objects|.
+            1.  If |options|[|key|] <a for="map">exists</a>, <a for="set">append</a> |object| to
+                |relevant interface objects|.
+
+        1.  Return |relevant interface objects|.
     </ol>
 
     ISSUE: jyasskin@ suggests replacing the iteration through the interface objects with a registry.
     I'm not sure which is less clear, honestly. I'll leave it like this for the moment, and we can
-    argue about whether this is too much of a `COMEFROM` interface.
+    argue about whether this is too much of a `COMEFROM` interface. (The addition of the
+    [[#sctn-registry-cred-type-to-options-member-identifier|Credential Type to Options Member Identifier Mappings]]
+    registry does not obviate this issue.)
   </div>
 
   <div algorithm="can collect locally">
-    A given {{CredentialRequestOptions}} |options| is
+    A given {{CredentialRequestOptions}} (|options|) is
```

Comment:
We don't usually parenthesize the variable name in algorithm headings like this. See https://infra.spec.whatwg.org/#algorithm-params.

Reply:
The credman spec presently parenthesizes the variable name in algorithm intro paragraphs (what you're referring to as "headings"). I we wish to change that let's please do it in a separate PR.

Reply:
Good point; fine with me.